### PR TITLE
Fix init docs formatting

### PR DESCRIPTION
**Pull request by zoran995**
### What this PR does

Fixes #<insert issue number here if relevant>

Fix formatting of doc for init files, some line breaks were missing.

### Checklist

-   [x] There are unit tests to verify my changes are correct or unit tests aren't applicable (if so, write quick reason why unit tests don't exist) - **_only docs change_**
-   [ ] I've updated CHANGES.md with what I changed.

### CODE REVIEW

File: doc/customizing/initialization-files.md [hunk 1]
```diff
@@ -76,8 +76,8 @@ Catalog files can be edited two ways:
 |stories|no|[**Story[]**](#story)||An array of stories to be loaded.|
 |viewerMode|no|**"3d"** or **"3dSmooth"** or **"2D"** |"3d"|The id of the viewer mode to be shown initialy.|
 |homeCamera|yes|[**CameraPosition**](#CameraPosition)||Where the camera goes when you click the "home" button between the zoom-in and zoom-out buttons.|
-|<a id="base-maps"></a>baseMaps|no|[**`baseMaps`**](#baseMaps)|[default list of basemaps](../../lib/Models/BaseMaps/defaultBaseMaps.ts)|The array of the base maps to be shown to the user.|
-|<a id="base-map-id"></a>baseMapId|no|**string**||The id of the baseMap user will see on the first mapLoad. The value must be an id of the catalog item from the [`baseMaps`](#base-maps) array|
+|<a id="base-maps"></a>baseMaps|no|[**`baseMaps`**](#baseMaps)|[default list of basemaps](https://github.com/TerriaJS/terriajs/blob/next/lib/Models/BaseMaps/defaultBaseMaps.ts)|The array of the base maps to be shown to the user.|
```
Comment (zoran995) [inline doc/customizing/initialization-files.md:79]: Have to use an absolute URL, hosted docs can't resolve relative URL to the repository.

Comment (na9da) [reply doc/customizing/initialization-files.md:79]: Good catch! We just need to remember to change the branch here once we have switched the branches.

File: doc/customizing/initialization-files.md [hunk 2]
```diff
@@ -76,8 +76,8 @@ Catalog files can be edited two ways:
...
+|<a id="base-map-id"></a>baseMapId|no|**string**||The id of the baseMap user will see on the first mapLoad. The value must be an id of the catalog item from the [`baseMaps`](#basemaps) array|
```
Comment (na9da) [inline doc/customizing/initialization-files.md]: Should we link it to the `#base-map-id` instead, just like the line below for `previewBaseMapId`?

Comment (zoran995) [reply doc/customizing/initialization-files.md]: I am not sure if I got the question, `previewBaseMapId` is linked to `base-map-id` (the line you tagged) for default value if it is not defined. You probably meant `base-maps` with a dash, yeah it should.

Comment (zoran995) [reply doc/customizing/initialization-files.md]: updated it to `base-maps`

File: doc/customizing/initialization-files.md (hunk 1 above)
Comment (AnaBelgun) [reply doc/customizing/initialization-files.md:79]: @na9da would you please submit a ticket and place it top of backlog so we don't forget when we switch to make this change?

**Comment by zoran995** [comment]
this is superseded by #5644 since base map definition is completely replaced, this issue is also resolved there

**Review by na9da — APPROVED** [review]

**Pull request merged**